src/hyperlight_guest/src/lib.rs

```diff
@@ -75,14 +67,17 @@ pub(crate) static _fltused: i32 = 0;
 #[allow(dead_code)]
 fn panic(info: &core::panic::PanicInfo) -> ! {
     unsafe {
-        let peb_ptr = P_PEB.unwrap();
         copy_nonoverlapping(
             info.to_string().as_ptr(),
-            (*peb_ptr).guestPanicContextData.guestPanicContextDataBuffer as *mut u8,
-            (*peb_ptr).guestPanicContextData.guestPanicContextDataSize as usize,
+            (*PEB)
+                .get_guest_panic_context_address()
+                .expect("Failed to get panic context") as *mut u8,
+            (*PEB)
```

Comment:
I think `count` should be the length of the string, not the length of the buffer. (Previous bug I know!)